core/src/main/java/com/linecorp/armeria/client/retry/RetryConfigMapping.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2020 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.client.retry;
+
+import java.util.function.BiFunction;
+
+import com.linecorp.armeria.client.ClientRequestContext;
+import com.linecorp.armeria.common.Request;
+import com.linecorp.armeria.common.Response;
+
+/**
+ * Returns a {@link RetryConfig} given the request context.
+ * Allows users to change retry behavior according to any context element, like host, method, path ...etc.
+ */
+@FunctionalInterface
+public interface RetryConfigMapping<T extends Response> {
+    /**
+     * Creates a {@link KeyedRetryConfigMapping} that maps keys created by {@code keyFactory} to
```

Comment:
```suggestion
     * Creates a {@link RetryConfigMapping} that maps keys created by {@code keyFactory} to
```
Because `KeyedRetryConfigMapping` is an internal class that is not exposed by Javadoc.

Reply:
Done